aviary/subsystems/aerodynamics/aerodynamics_builder.py

```diff
@@ -17,8 +17,8 @@
 from aviary.subsystems.aerodynamics.gasp_based.premission_aero import PreMissionAero
 from aviary.subsystems.aerodynamics.gasp_based.gaspaero import CruiseAero
 from aviary.subsystems.aerodynamics.gasp_based.gaspaero import LowSpeedAero
-from aviary.subsystems.aerodynamics.gasp_based.table_based import CruiseAero as TabularCruiseAero
-from aviary.subsystems.aerodynamics.gasp_based.table_based import LowSpeedAero as TabularLowSpeedAero
+from aviary.subsystems.aerodynamics.gasp_based.table_based import TabularCruiseAero
+from aviary.subsystems.aerodynamics.gasp_based.table_based import TabularLowSpeedAero as TabularLowSpeedAero
```

Comment:
You don't need the import alias, since the component name changed to what we want.